src/main/java/rest/CompletableFutureService.java

```diff
@@ -0,0 +1,18 @@
+package rest;
+
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import java.io.IOException;
+import java.net.http.HttpResponse;
+
+public class CompletableFutureService {
+    public static void main(String[] args) throws IOException {
```

Comment:
do we need this class? can we replace it for unit test?

Reply:
Class was removed.